src/main/java/net/sf/jabref/logic/importer/fileformat/ModsImporter.java

```diff
@@ -187,7 +185,7 @@ private void parseModsGroup(Map<String, String> fields, List<Object> modsGroup,
         }
 
         //The element subject can appear more than one time, that's why the keywords has to be put out of the for loop
-        putIfListIsNotEmpty(fields, keywords, FieldName.KEYWORDS, KEYWORD_SEPARATOR.toString());
+        putIfListIsNotEmpty(fields, keywords, FieldName.KEYWORDS, KEYWORD_SEPARATOR);
```

Comment:
the keywords were changed from ", " to a char ','. Thats why the test won't work. This wasn't detected because the test weren't running.